zubhub_frontend/zubhub/src/assets/js/icons/projectsCountIcon.js

```diff
@@ -5,7 +5,7 @@ function ProjectsCountIcon() {
     <svg xmlns="http://www.w3.org/2000/svg" 
       width="1.48em"
       height="1.2em"
-      fill="#00B8C4"
+      fill="var(--primary-color3)"
```

Comment:
I don't think css variables work in html attributes? I haven't tested it.
Can you check and use context if it doesn't work